src/ontology/pato-edit.obo

```diff
@@ -21847,6 +21847,40 @@ def: "A degree of pigmentation quality that is diversified with patches, or irre
 is_a: PATO:0002248 ! pigmented
 created_by: https://orcid.org/0000-0002-3528-5267
 
+[Term]
+id: PATO:0070002
+name: basket cell morphology
+def: "A cell morphology that inheres in neurons that have sparsely spinous dendrite, which spread radially and tangentially, with axons on the pial side of the soma ascending or descending to give off horizontal collaterals, which in turn sprout small branches ending in dense tufts of terminals." [doi:10.1016/b978-0-12-369497-3.10004-4]
+xref: ILX:0101123
+xref: ilxtr:BasketPhenotype
+xref: nifext:3
+is_a: PATO:0010006 ! cell morphology
+created_by: http://orcid.org/0000-0001-7258-9596
+
+[Term]
+id: PATO:0070003
+name: small basket cell morphology
+def: "A basket cell morphology that inheres in neurons that are large, aspiny and multipolar, and gives rise to many long horizontally and vertically projecting axon collaterals that traverse neighbouring columns." [PMID:11884355]
```

Comment:
This one is a bit difficult for me to parse. What is it that "gives rise to many long horizontally and vertically projecting axon collaterals"? The morphology?

Reply:
my bad, I've gone to the paper and copied over the correct def for this